docs/articles/building-source.md

```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
+2. Clone the [repository](https://github.com/files-community/files) and open `Files.sln` in VS
+3. Make sure that `Files.Package` is set as the startup item
```

Comment:
Can you provide instructions on how someone can set the correct startup project?

Reply:
what do you mean?

Reply:
Step 3 is to set Files.Package as the startup item, can you provide instructions on how to do that?

Reply:
Done!